### PR TITLE
Include body in case of BodyResource

### PR DESCRIPTION
**Pull request by wouterj**
This allows to get the contents of the files from the file system.

/cc @ElectricMaxxx

### CODE REVIEW

**Comment by dbu** [comment]
there is a build error, looks like it has to do with hhvm and not related to the PR

**Comment by ElectricMaxxx** [comment]
@WouterJ cool. Is there a chance have a getAll() route to get a collection of snippets?

**Comment by wouterj** [comment]
@ElectricMaxxx yes, by requesting `/api/snippets/` (i.e. the root directory), you get a directory resource as response, this resource has a `children` option with the files in that directory (and if you increase the depth, you can also make it show files inside subdirectories).

**Comment by ElectricMaxxx** [comment]
That is absolutely awesome.

File: Tests/Features/Context/ResourceContext.php [hunk 1]
```diff
@@ -17,6 +17,9 @@ class ResourceContext implements Context, KernelAwareContext
 {
     private $session;
     private $manager;
+    /**
```
Comment (dantleech) [inline Tests/Features/Context/ResourceContext.php:21]: I think there should be an empty line before a doc block.

**Comment by wouterj** [comment]
ping @dantleech ready to merge?

**Comment by dantleech** [comment]
Yep looks good to me, merging.

**Pull request merged**